seahorn/jobs/array_list_back/aws_bounded_array_list_back_harness.c

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ */
+
+#include <seahorn/seahorn.h>
+#include <aws/common/array_list.h>
+#include <bounded_array_list_helper.h>
+#include <utils.h>
+#include <proof_allocators.h>
+
+/**
+ * Runtime: 19s
+ */
+int main() {
+
+    /* data structure */
+    struct aws_array_list list;
+    initialize_bounded_array_list(&list, MAX_BLOCK_SIZE);
```

Comment:
also make this work with seahorn

Reply:
Do you mean to make this harness passed by running seahorn?